Tools/Scripts/libraries/webkitscmpy/webkitscmpy/local/git.py

```diff
@@ -74,10 +74,7 @@ def path(self):
 
         def _fill(self, branch):
             default_branch = self.repo.default_branch
-            if branch == default_branch:
-                branch_point = None
-            else:
-                branch_point = int(self._hash_to_identifiers[self._ordered_commits[branch][0]].split('@')[0])
```

Comment:
Right, it seems https://github.com/WebKit/WebKit/pull/1896 solved this issue since I can reproduce the bug on one commit before e316c781f551 but not e316c781f551.